lib/config/rules/stylistic-issues.js

```diff
@@ -1,8 +1,12 @@
 // see http://eslint.org/docs/rules/#stylistic-issues
 
 module.exports = {
+  // enforce linebreaks after opening and before closing array brackets <FEEDBACK>
+  'array-bracket-newline': ['error', {'multiline': true}],
```

Comment:
No need for quotes when names are valid identifiers

Reply:
Disabling this rule, it's not behaving as expected, or maybe I'm misunderstanding it. 

Lint errors show that there should be no linebreak when we specify `multiline: true`. 
https://circleci.com/gh/Shopify/eslint-plugin-shopify/144